benchto-driver/src/main/java/com/teradata/benchto/driver/loader/BenchmarkLoader.java

```diff
@@ -280,7 +283,10 @@ private void evaluateValueExpressions(Map<String, String> variablesMap)
 
     private String benchmarkName(Path benchmarkFile)
     {
-        String relativePath = properties.benchmarksFilesPath().relativize(benchmarkFile).toString();
+        Path benchmarkFilesDir = properties.benchmarksFilesDirs().stream()
+                .filter(benchmarkFile::startsWith)
+                .findFirst().get();
```

Comment:
this will lead to duplicate benchmark names which will be caught when loading benchmarks.